packages/braid-design-system/src/lib/components/MenuRenderer/MenuRenderer.screenshots.tsx

```diff
@@ -34,13 +36,41 @@ const defaultProps = {
 
 const triggerHeight = 44;
 
-const triggerPosition = {
-  top: triggerHeight,
-  left: 0,
-  bottom: 0, // this value is ignored when placement is top
-  right: 0, // this value is ignored when align is left
+const PopoverWrapper = ({
+  children,
+  popoverPlacement,
+}: {
+  children: (props: {
+    triggerRef: MutableRefObject<HTMLButtonElement | null>;
+  }) => ReactNode;
+  popoverPlacement: PopoverProps['placement'];
+}) => {
+  const triggerRef = useRef<HTMLButtonElement | null>(null);
+  const wrapperPadding = calc(vars.touchableSize).multiply(2.5).toString();
+
+  return (
+    <Box
+      display="flex"
+      style={
+        popoverPlacement === 'bottom'
+          ? { paddingBottom: wrapperPadding }
+          : { paddingTop: wrapperPadding }
+      }
+    >
+      <Box ref={triggerRef}>
+        <Placeholder label="Menu trigger" height={triggerHeight} />
+      </Box>
+      {children({ triggerRef })}
+    </Box>
+  );
 };
 
+const ScreenshotPopover = ({
+  ...props
+}: Omit<PopoverProps, 'open' | 'lockPlacement' | 'role'>) => (
+  <Popover open lockPlacement role={false} {...props} />
+);
```

Comment:
Lets move this from a component abstraction to a default props object that can be shared between examples below.